File: connectors/connector-basecamp/src/test/scala/com/taskadapter/connector/basecamp/BasecampFieldBuilder.scala
package com.taskadapter.connector.basecamp

import com.taskadapter.connector.FieldRow

object BasecampFieldBuilder {
  def getDefault(): List[FieldRow[_]] = {
    List(
      FieldRow.apply(BasecampField.content, BasecampField.content, ""),
    )
  }
}

File: connectors/connector-basecamp/src/test/java/com/taskadapter/connector/basecamp/BasecampFieldBuilder.java
package com.taskadapter.connector.basecamp;

import com.taskadapter.connector.FieldRow;

import java.util.List;

public class BasecampFieldBuilder {
    static List<FieldRow<?>> getDefault() {
        return List.of(
                FieldRow.apply(BasecampField.content, BasecampField.content, "")
        );
    }
}
